indico/modules/events/registration/templates/management/regform_list.html

```diff
@@ -5,6 +5,30 @@
     {%- trans %}Registration{% endtrans -%}
 {% endblock %}
 
+{% block title_actions %}
+    <a class="i-button inconspicuous icon-settings arrow js-dropdown"
+       data-toggle="dropdown"></a>
+    <ul class="dropdown">
+        <li>
+            <a href="#"
+               title="{% trans %}Add/Remove users allowed to modify registration{% endtrans %}"
+               data-title="{% trans %}Manage registration managers{% endtrans %}"
+               data-href="{{ url_for('.manage_registration_managers', event) }}"
+               data-qtip-position="right"
+               data-ajax-dialog>
+                {% trans %}Manage registration managers{% endtrans %}</a>
+        </li>
+        {% if event.getType() == 'conference' %}
```

Comment:
yep, i'd change the code calling this template to pass the event_new as `event` and then use `event.type` instead. any other usages of `event` in the template should already work fine with an Event object instead of a Conference